### PR TITLE
Media overlay fragments

### LINKED ISSUE #1586: Is reference to xpointer shorthand for fragment identifiers too restrictive?
The media overlay attributes that expect a fragment identifier say they require: "a fragment identifier that references the specific element as per the [XPTRSH]".

Is the reference to xpointer shorthand necessary here? If we were to actually follow XPTRSH we'd invalidate many potential ID references into HTML documents as HTML IDs no longer have to be NCName values.

Doesn't referencing XPTRSH also negate being able to use EPUBCFI?

### PR DESCRIPTION
**Pull request by mattgarrish**
Here's my best attempt at resolving #1586.

I've added a new section to the authoring spec specifically dealing with fragment identifiers and have linked down to it from the textref and src attributes.

I've specified that authors should reference a target element or use an svg fragid, but may use other schemes (noting that support is not required).

In the reading system specification, I've made the recommendation to ensure the element is visible contingent on one of the above two fragment identifier schemes being used.

Otherwise, I've made some minor cleanup to ensure we're not being too element-specific when talking about the fragments (plus some general cleanup since I was reading through the sections anyway).

Fully expect this will need some more tweaking, though. I'll add a change log entry once it's clear what we're settling on.

Fixes #1586

- Reading Systems [preview](https://cdn.statically.io/gh/w3c/epub-specs/fix/issue-1586/epub33/rs/index.html)
- Reading Systems [diff](https://services.w3.org/htmldiff?doc1=https://labs.w3.org/spec-generator/%3Ftype=respec%26url=https://w3c.github.io/epub-specs/epub33/rs/index.html&doc2=https://labs.w3.org/spec-generator/%3Ftype=respec%26url=https://cdn.statically.io/gh/w3c/epub-specs/fix/issue-1586/epub33/rs/index.html)

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/epub-specs/pull/1663.html" title="Last updated on May 6, 2021, 12:31 PM UTC (5d6562e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/epub-specs/1663/765799e...5d6562e.html" title="Last updated on May 6, 2021, 12:31 PM UTC (5d6562e)">Diff</a>

### CODE REVIEW

**Review by marisademeglio — APPROVED** [review]

**Review by dauwhe — APPROVED** [review]
I like that this references HTML and (indirectly) the URL spec.

File: epub33/core/index.html [hunk 1]
```diff
@@ -7247,20 +7244,43 @@ <h5>Overlay Structure</h5>
...
+						<p>The <code>epub:textref</code> attribute and the <a href="#elemdef-smil-text"
+									><code>text</code> element's</a>
+							<code>src</code> attribute both require a fragment identifier that references a specific
+							fragment (e.g., an element or text string) of the associated <a>EPUB Content
+							Document</a>.</p>
+
+						<p>For XHTML and SVG Content Documents, the fragment SHOULD be a reference to a <a
+								href="https://html.spec.whatwg.org/multipage/browsing-the-web.html#target-element"
+								>target element</a> [[HTML}] or an <a
+								href="https://www.w3.org/TR/SVG2/linking.html#SVGFragmentIdentifiers">SVG Fragment
```
Comment (iherman) [inline epub33/core/index.html:7259]: [[HTML}] -> [[HTML]]

File: epub33/core/index.html [hunk 2]
```diff
@@ -6805,8 +6805,8 @@ <h5>The <code>seq</code> Element</h5>
 									</dt>
 									<dd>
 										<p>The relative IRI reference [[RFC3987]] of the corresponding EPUB Content
-											Document, including a fragment identifier that references the specific
-											element as per the [[XPTRSH]].</p>
+											Document, including a <a href="sec-media-overlays-fragids">fragment
+												identifier</a> that references the specific content.</p>
```
Comment (iherman) [inline epub33/core/index.html:6809]: sec-media-overlays-fragids -> #sec-media-overlays-fragids

Comment (iherman) [inline epub33/core/index.html:6809]: sec-media-overlays-fragids -> #sec-media-overlays-fragids

File: epub33/core/index.html [hunk 3]
```diff
@@ -6937,8 +6937,8 @@ <h5>The <code>text</code> Element</h5>
...
```
Comment (iherman) [inline epub33/core/index.html:6941]: sec-media-overlays-fragids -> #sec-media-overlays-fragids

**Review by iherman — APPROVED** [review]

File: epub33/core/index.html (hunk 3 above)
Comment (mattgarrish) [reply epub33/core/index.html:6941]: Oops, ya, bad copy/paste job.

**Review by wareid — APPROVED** [review]

**Pull request merged**